src/api/world.ts

```diff
@@ -246,6 +246,7 @@ export const findWorldBySlug = async ({
   const worldsRef = await firebase
     .firestore()
     .collection(COLLECTION_WORLDS)
+    .where("isHidden", "==", false)
```

Comment:
Considering `isHidden` is a new field, more prudent action would be to count `undefined`  same as `false`. This also eliminates the need to run a migration script to "unhide"...

On second thought, why not just `isVisible` and set to `true` only for those that should be seen?

Reply:
Firestore rules don't play nice with undefined fields. At all. I spent hours trying to get it to work :(

Reply:
This is why I've also suggested `!=true` as an option elsewhere or turning the tables from `isHidden` to `isVisible`, but at the end of the day, I'm just providing few different opinions for good measure and will leave it up to your best judgement and possibly @mike-lvov's second review after he'd already approven